sdk/eventgrid/azure-eventgrid/tests/test_exceptions_async.py

```diff
@@ -4,31 +4,20 @@
 # license information.
 #--------------------------------------------------------------------------
 
-import logging
-import sys
 import os
 import json
 import pytest
-import uuid
-from datetime import datetime, timedelta
-from azure.core.exceptions import HttpResponseError, ClientAuthenticationError, ServiceRequestError
+from azure.core.exceptions import HttpResponseError, ClientAuthenticationError
 from msrest.serialization import UTC
```

Comment:
similar to sync does this import also need to go?

Reply:
no its used in the tests